docs/rules/indent-legacy.md

```diff
@@ -1,5 +1,6 @@
 # enforce consistent indentation (indent-legacy)
 
+This rule was **deprecated** in ESLint v4.0.0.
```

Comment:
Should have blank line after: 
```suggestion
This rule was **deprecated** in ESLint v4.0.0.

```

Reply:
Done https://github.com/eslint/eslint/pull/14989/commits/48b21c2e06d98bc8aed0a067193209ab52c760c5